src/common/layout.marko

```diff
@@ -12,7 +12,7 @@ $ const i18n = out.stream.req.t;
 
   <!-- GOV.UK header link text -->
   <@global-header-text>
-    <abbr title="The Department of Work and Pensions">DWP</abbr>
+    <abbr title="The Department of Work and Pensions Digital">DWP Digital</abbr>
```

Comment:
Since digital isn't part of the acronym, I'd have done:
```html
<abbr title="The Department of Work and Pensions">DWP</abbr> Digital
```
Thoughts?